examples/nft/scripts/transfer/common.js

```diff
@@ -26,11 +27,18 @@ exports.printLocalNFT = async function (deployer, account, appId) {
     try {
         const localState = await readLocalStateSSC(deployer, account, appId);
         // each nft is stored as a one record in user store
-        let count = 0;
-        if (localState !== undefined)
-            count = localState.length
-        console.log('NFT balance of', account, ':', count);
+        let holdings = "";
+        if (localState === undefined) {
+            holdings = "null";
+        }
+        else {
+            for(const l of localState) {
+                const key = decode(Buffer.from(l.key, 'base64'));
+                holdings += key.toString() + ' ';
+            }
+        }
+        console.log('NFT holdings of', account, ':', holdings);
```

Comment:
can you show an example output please?

Reply:
Sure.
- Before Transfer
![image](https://user-images.githubusercontent.com/33264364/100930090-97f28600-350e-11eb-8dfd-4ec3fe531528.png)
- After Transfer
![image](https://user-images.githubusercontent.com/33264364/100930119-a50f7500-350e-11eb-8ff7-4159b298922d.png)

Reply:
thanks. I will update the message text.